docs/domains/Domain.md

```diff
@@ -193,6 +196,21 @@ ServerPod describes the configuration for a Kubernetes pod for a server.
 | `annotations` | Map | The annotations to be attached to generated resources. |
 | `labels` | Map | The labels to be attached to generated resources. The label names must not start with 'weblogic.'. |
 
+### Model
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `configMap` | string | WDT config map name. Required. |
+| `domainType` | string | WDT domain type: Legal values: WLS, RestrictedJRF, JRF. Defaults to WLS. |
+| `encryptionSecret` | [Secret Reference](k8s1.13.5.md#secret-reference) | WDT encryption key passphrase secret. Required when WDT model files are encrypted. |
+
+### Opss
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `walletFileSecret` | [Secret Reference](k8s1.13.5.md#secret-reference) | Secret containing the OPSS key wallet file. |
+| `walletPasswordSecret` | [Secret Reference](k8s1.13.5.md#secret-reference) | Secret containing OPSS key passphrase. |
```

Comment:
What is an '(k8s1.13.5.md#secret-reference)', simply a secret name? Can we support just putting a secret name here?

Reply:
Change to just a string.